[2023-12-25 21:03:47,680] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: titanic_pivot.start manual__2023-12-25T21:03:46.355583+00:00 [queued]>
[2023-12-25 21:03:47,688] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: titanic_pivot.start manual__2023-12-25T21:03:46.355583+00:00 [queued]>
[2023-12-25 21:03:47,688] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-12-25 21:03:47,688] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-12-25 21:03:47,689] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-12-25 21:03:47,700] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): start> on 2023-12-25 21:03:46.355583+00:00
[2023-12-25 21:03:47,704] {standard_task_runner.py:52} INFO - Started process 40047 to run task
[2023-12-25 21:03:47,707] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'titanic_pivot', 'start', 'manual__2023-12-25T21:03:46.355583+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/titanic_dag.py', '--cfg-path', '/tmp/tmpvnwvlur1', '--error-file', '/tmp/tmphzdxvtzg']
[2023-12-25 21:03:47,708] {standard_task_runner.py:80} INFO - Job 22: Subtask start
[2023-12-25 21:03:47,775] {task_command.py:369} INFO - Running <TaskInstance: titanic_pivot.start manual__2023-12-25T21:03:46.355583+00:00 [running]> on host c975c50309df
[2023-12-25 21:03:47,851] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=start
AIRFLOW_CTX_EXECUTION_DATE=2023-12-25T21:03:46.355583+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-25T21:03:46.355583+00:00
[2023-12-25 21:03:47,851] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-25 21:03:47,852] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Here we start! "']
[2023-12-25 21:03:47,859] {subprocess.py:85} INFO - Output:
[2023-12-25 21:03:47,860] {subprocess.py:92} INFO - Here we start!
[2023-12-25 21:03:47,860] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-25 21:03:47,897] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=start, execution_date=20231225T210346, start_date=20231225T210347, end_date=20231225T210347
[2023-12-25 21:03:47,920] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-25 21:03:47,964] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
